ShoppingCart/ShoppingCart/Cart.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System.Collections.Generic;
+
+namespace ShoppingCart
+{
+    public class Cart
+    {
+        public List<CartItem> CartItemList{ get; private set; }
+        public double TotalCartCost { get; private set; }
+
+        public double DiscountedTotalCost { get; private set; }
+
+        private Discounts _discounts;
+
+        public Cart()
+        {
+                // Constructor for the Cart objects

```

Comment:
Unnecessary Comment! (as it was obvious that this was a constructor)
Check Shared "Shared Clean Code Doc"for Commenting Rules!